yaptide/batch/string_templates.py

```diff
@@ -34,14 +56,23 @@
 ROOT_DIR={root_dir}
 INPUT_WILDCARD=$ROOT_DIR/workspaces/task_*/*.bdo
 OUTPUT_DIRECTORY=$ROOT_DIR/output
-
-cd $ROOT
+BIN_DIR=$ROOT_DIR/bin
 
 mkdir -p $OUTPUT_DIRECTORY
 
+cd $OUTPUT_DIRECTORY
+
 for INPUT_FILE in $INPUT_WILDCARD; do
-  cp $INPUT_FILE $OUTPUT_DIRECTORY
+    mv $INPUT_FILE $OUTPUT_DIRECTORY
 done
+
+$BIN_DIR/convertmc json --many "$OUTPUT_DIRECTORY/*.bdo"
```

Comment:
You can do as well:
$BIN_DIR/convertmc json --many "$INPUT_WILDCARD"

and remove moving all bdo files to the $OUTPUT directory.

For debugging purposes its more convenient if BDO files stays where they have been generated